config/400-webhook-service.yaml

```diff
@@ -16,12 +16,12 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    role: eventing-webhook
-  name: eventing-webhook
+    role: webhook
+  name: webhook
   namespace: knative-eventing
 spec:
```

Comment:
Why are the Service and the Deployment for the webhook in separate files?

Can we combine these at the 400 level?